metal/datasource_metal_port.go

```diff
@@ -1,15 +1,12 @@
 package metal
 
 import (
-	"fmt"
-
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
-	"github.com/packethost/packngo"
 )
 
 func dataSourceMetalPort() *schema.Resource {
 	return &schema.Resource{
-		Read: dataSourceMetalPortRead,
+		Read: resourceMetalPortRead,
 
```

Comment:
The `Import` resource should work at this stage of development too. And we can add a test for that.